### ISSUE TITLE
Intercom makes the application crash with any declared GlideModule 

### ISSUE THREAD

**Issue by Sefford** [intercom/intercom-android#429]
## Version info
  - Intercom for Android version: 4.0.4
  - Android OS version: N/A
  - Android Device: N/A

## Expected behavior

Application should be able to access Intercom UI/notification normally.

## Actual behavior

Application crashes when attempting to access any Intercom UI (notifications, conversation lists...).

## Steps to reproduce
 1.  Use Glide library (current version: 3.8.0)
 2.  Declare at least a GlideModule on your AndroidManifest.xml
 3. Attempt to access any kind of Intercom UI (Conversation lists, notifications, messaging...)

## Logs
```
java.lang.RuntimeException: Expected instanceof GlideModule, but found: com.bumptech.glide.integration.okhttp3.OkHttpGlideModule@d24cf6c
                                                                 at io.intercom.com.bumptech.glide.module.ManifestParser.parseModule(ManifestParser.java:81)
                                                                 at io.intercom.com.bumptech.glide.module.ManifestParser.parse(ManifestParser.java:43)
                                                                 at io.intercom.com.bumptech.glide.Glide.initializeGlide(Glide.java:193)
                                                                 at io.intercom.com.bumptech.glide.Glide.checkAndInitializeGlide(Glide.java:172)
                                                                 at io.intercom.com.bumptech.glide.Glide.get(Glide.java:156)
                                                                 at io.intercom.com.bumptech.glide.Glide.getRetriever(Glide.java:535)
                                                                 at io.intercom.com.bumptech.glide.Glide.with(Glide.java:561)
                                                                 at io.intercom.android.sdk.utilities.AvatarUtils.createAvatar(AvatarUtils.java:63)
                                                                 at io.intercom.android.sdk.utilities.AvatarUtils.loadAvatarIntoView(AvatarUtils.java:44)
                                                                 at io.intercom.android.sdk.views.AuthorAvatarView.loadAvatarWithActiveState(AuthorAvatarView.java:125)
                                                                 at io.intercom.android.sdk.inbox.InboxRowViewHolder.setRowParticipantDetails(InboxRowViewHolder.java:93)
                                                                 at io.intercom.android.sdk.inbox.InboxRowViewHolder.bindData(InboxRowViewHolder.java:64)
                                                                 at io.intercom.android.sdk.inbox.InboxAdapter.onBindViewHolder(InboxAdapter.java:69)
                                                                 at android.support.v7.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:6482)

[Follows, but it is no longer relevant]
```

## Additional info

Intercom works up to 4.0.3, and it happens to any type of module.

**Comment by ronocod** [comment]
Thanks for reporting this, we'll be releasing a new version today that will fix this issue.

**Comment by Sefford** [comment]
Thanks guys! :+1:

**Comment by ronocod** [comment]
Fixed in 4.0.5.
